packages/edit-site/src/components/editor/global-styles-renderer.js

```diff
@@ -116,8 +116,8 @@ function flattenTree( input = {}, prefix, token ) {
 function getStylesDeclarations( blockStyles = {} ) {
 	return reduce(
 		STYLE_PROPERTY,
-		( declarations, { value, valueGlobal, properties }, key ) => {
-			const pathToValue = valueGlobal ?? value;
+		( declarations, { value, properties }, key ) => {
```

Comment:
We can remove more bytes from this :hocho: :hocho: :hocho: (remove the logic that processed keys starting with `--`). We also need to prevent processing the `linkColor` property (properties whose value starts by `elements`).